daemon/graphdriver/devmapper/driver.go

```diff
@@ -160,8 +160,9 @@ func (d *Driver) Remove(id string) error {
 // Get mounts a device with given id into the root filesystem
 func (d *Driver) Get(id, mountLabel string) (string, error) {
 	mp := path.Join(d.home, "mnt", id)
+	rootFs := path.Join(mp, "rootfs")
 	if count := d.ctr.Increment(mp); count > 1 {
```

Comment:
@cpuguy83 I think so, `mp: /var/lib/docker/devicemapper/mnt/{ID}`  is still the mountpoint and should be the right path to ref counter.